src/bin/cargo/commands/metadata.rs

```diff
@@ -11,8 +11,8 @@ pub fn cli() -> Command {
         )
         .arg(multi_opt(
             "filter-platform",
-            "TRIPLE",
-            "Only include resolve dependencies matching the given target-triple",
+            "TRIPLE|host-tuple|host",
```

Comment:
I don't think `host-tuple` is important enough to add to help text.
(At least other commands didn't)

Let's treat them the same.